src/components/AnonymousChat/child/ChatReplyPreview.tsx

```diff
@@ -27,15 +29,33 @@ const ChatReplyPreview = ({type}: ChatReplyPreviewProps) => {
       exiting={SlideOutDown.duration(80)}
       style={styles.container}>
       <View style={textContainerStyle}>
-        <View style={styles.chatTitleContainer}>
-          <Text style={styles.userText}>{replyPreview?.username}</Text>
-          <View style={styles.dot} />
-          <Text style={styles.timeText}>{replyPreview?.time}</Text>
-        </View>
+        <View>
+          <View style={styles.chatTitleContainer}>
+            <Text style={styles.userText}>{replyPreview?.username}</Text>
+            <View style={styles.dot} />
+            <Text style={styles.timeText}>{replyPreview?.time}</Text>
+          </View>
 
-        <Text style={styles.text} numberOfLines={1}>
-          {`${replyPreview?.message}`}
-        </Text>
+          <Text
+            style={[styles.text, replyPreview?.attachments.length > 0 ? {fontStyle: 'italic'} : {}]}
+            numberOfLines={1}>
+            {replyPreview?.attachments.length > 0 ? 'Photo' : `${replyPreview?.message}`}
+          </Text>
+        </View>
+        {replyPreview?.attachments.length > 0 && (
+          <View style={styles.mediaContainer}>
+            <FastImage
+              source={{uri: replyPreview?.attachments[0]?.asset_url}}
+              style={styles.mediaImage}
+              resizeMode="cover"
+            />
+            {replyPreview?.attachments[0].video_path && (
+              <View style={styles.mediaOverlay}>
+                <IconVideoPlay width={24} height={24} />
+              </View>
+            )}
+          </View>
+        )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code checks if `replyPreview?.attachments.length > 0` multiple times. It would be more efficient to store this in a variable and use it instead. Also, there is no null check for `replyPreview`. If `replyPreview` is null or undefined, accessing `replyPreview.attachments` will throw an error.
```diff
- {replyPreview?.attachments.length > 0 ? 'Photo' : `${replyPreview?.message}`}
+ const hasAttachments = replyPreview?.attachments?.length > 0;
+ {hasAttachments ? 'Photo' : `${replyPreview?.message}`}

- {replyPreview?.attachments.length > 0 && (
+ {hasAttachments && (
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->